packages/client/src/components/room.tsx

```
@@ -8,75 +8,90 @@
+  error?: string;
+}
 
-type ExtraContentFields = {
-  name: string;
-  createdAt: number;
-  isLoading?: boolean;
-};
-
-type ContentWithUser = Content & ExtraContentFields;
+// Local definition for ParsedMediaInfo
+interface ParsedMediaInfo {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ParsedMediaInfo'. Consider to delete it or rename it.

'ParsedMediaInfo' is defined here:


(lint/suspicious/noRedeclare)